test-foundry/helpers/ConcreteThreeHeapOrdering.sol

```diff
@@ -13,6 +13,10 @@ contract ConcreteThreeHeapOrdering is ICommonHeapOrdering {
         return heap.accounts[_index].value;
     }
 
+    function accountsId(uint256 _index) external view returns (address) {
```

Comment:
`accountsValue` and `accountsId` are quite misleading since the returned value is linked to one single account, but clearly not a problem

Reply:
It has been done like this because of the name of the mapping. You would prefer `accountValue` and `accountId` right ?

Reply:
As `indexOf`, we also can use `idAtIndex` and `valueAtIndex`

Reply:
Idk `accountsId` and `accountsValue` were great too